### ISSUE TITLE
Non-ASCII characters from updates.csv in log files

### ISSUE THREAD

**Issue by jteresco** [TravelMapping/DataProcessing#372]
In #189, the issue has come up about non-ASCII characters from updates.csv entries now getting into user .log files, and not displaying properly either when generated by Python or served by Apache on FreeBSD.

It seems the .log files should be generated with a UTF-8 encoding.

**Comment by yakra** [comment]
> It seems the .log files should be generated with a UTF-8 encoding.

I tried slapping a UTF-8 byte order mark at the beginning of a .log file, and serving it from yakra.teresco.org. It was a no go.

> when generated by Python

Python & C++ flavored logs are the same.
```sh
diff <(tail -n +2 python-teresco/logs/users/$username.log) <(tail -n +2 cplusplus/logs/users/$username.log)
```
produces no output.

> or served by Apache on FreeBSD

I think this is where the issue lies.

**Comment by yakra** [comment]
> It seems the .log files should be generated with a UTF-8 encoding.

If I slap a UTF-8 byte order mark at the beginning of the .log file, things look better:
http://yakra.teresco.org/logs/michih.orig.log
http://yakra.teresco.org/logs/michih.utf8.log
...but not perfect:
http://yakra.teresco.org/logs/oscar.orig.log
http://yakra.teresco.org/logs/oscar.utf8.log
(search for `montr`)
Hopefully once I have a go at canqca_con.csv *(also canqca.csv?)* with a hex editor this will clear up too.

**Comment by yakra** [comment]
@jteresco, What do you have for `AddDefaultCharset` in httpd.conf?

**Comment by yakra** [comment]
Non-ascii chars also appear in siteupdate.log when listing the commented-out systems.csv lines.

**Comment by yakra** [comment]
https://travelmapping.net/logs/users/michih.log

> 2016-08-08 | (Germany) Baden-WÃ¼rttemberg | SÃ¼dtangente Karlsruhe | deubw.sudtankar | New Route

**Comment by DrFrankenstein** [comment]
I think `AddDefaultCharset` is worth trying and probably the easiest fix, but some caveats:
- if pages in the site use an encoding other than UTF-8 (such as ISO-8859-1), this'll break them.
  - ...but IMO we should really be using UTF-8 everywhere.
- if some users write their config file in a coding other than UTF-8 (mainly in `# comments`), this might break browser rendering for those.
  - no real fix for this, but that's probably a really rare edge case, I haven't seen anyone other than me even using comments yet.
